Lapka.Pets.Application/Exceptions/PetNotFoundException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Security.AccessControl;
+
+namespace Lapka.Pets.Application.Exceptions
+{
+    public class PetNotFoundException : AppException
+    {
+        public Guid Id { get; }
+        public override string Code => "pet_not_found";
+
+        public PetNotFoundException(Guid id) : base($"Pet not exists in database: {id}")
```

Comment:
Dont write that in database. And i think that this message should be "Pet DOES not exist"